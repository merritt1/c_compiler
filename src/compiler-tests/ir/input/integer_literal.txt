int main(void) {
	int i;
	i = 5;
}