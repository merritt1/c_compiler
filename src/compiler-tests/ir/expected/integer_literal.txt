(procBegin,  main)
(addressOf,     r0,      i)
(constInt,     r1,          5)
(storeWord,     r0,      r1)
(procEnd ,  main)
